docs/reference_keymap_extras.md

```diff
@@ -4,7 +4,7 @@ Keyboards are able to support a wide range of languages. However, this support i
 
 Obviously, this can get confusing, so QMK provides language-specific keycode aliases for many keyboard layouts. These won't do much on their own - you still have to set the matching keyboard layout in your OS settings. Think of them more as keycap labels for your keymap.
 
-Simply `#include` one of the keycode headers below at the top of your `keymap.c`, and assign the keycodes defined in the header in place of the `KC_` prefixed ones.
+Simply `#include` one of the keycode headers below at the top of your `keymap.c`, or set the `host_language` field in your `keymap.json` to the corresponding language. Then assign the keycodes defined in the header in place of the `KC_` prefixed ones.
```

Comment:
````suggestion
Simply `#include` one of the keycode headers below at the top of your `keymap.c`, and assign the keycodes defined in the header in place of the `KC_` prefixed ones.

If your keymap is setup using a `keymap.json` file, this can be accomplished using the `host_language` key in that file. Example:
```json
{
    "keyboard": "handwired/my_macropad",
    "keymap": "my_keymap",
    "host_language": "swedish",
    "layout": "LAYOUT_all",
    "layers": [
        ["SE_A"]
    ]
}
```
````
An example will help remove ambiguity with the syntax.

Reply:
As the example already exists on the Macros page, should we just move it into here and reference it in macros? See https://github.com/qmk/qmk_firmware/pull/24584/commits/bbe17cfebb8653bb40769d25155b6716a86542f4